stderr of test 'tst161` in directory 'monetdb5/mal` itself:


# 19:09:44 >  
# 19:09:44 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=43514 --set sql_port=57655 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst161.mal </dev/null
# 19:09:44 >  


# 14:57:21 >  
# 14:57:21 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-5138" "--port=37326"
# 14:57:21 >  

MAPI  = (monetdb) /var/tmp/mtest-5138/.s.monetdb.37326
QUERY =     return l:= 0:mytype;
ERROR = !SyntaxException:defConstant:constant coercion error from int to mytype
MAPI  = (monetdb) /var/tmp/mtest-5138/.s.monetdb.37326
QUERY = end l;
ERROR = !SyntaxException:mytype.l[0]:RETURN missing
MAPI  = (monetdb) /var/tmp/mtest-5138/.s.monetdb.37326
QUERY = z:= mytype.l();
ERROR = !TypeException:user.main[1]:'mytype.l' undefined in:     z:any := mytype.l();
MAPI  = (monetdb) /var/tmp/mtest-5138/.s.monetdb.37326
QUERY = io.print(z);
ERROR = !SyntaxException:user.main[0]:RETURN missing

# 14:57:21 >  
# 14:57:21 >  "Done."
# 14:57:21 >  

